decentraland/unity-explorer
# Style: minimap animations and improvements

## What does this PR change?

This PR improves the minimap UI and adds animations.
In this spreadsheet you can find the different tasks achieved with the PR https://docs.google.com/spreadsheets/d/1K5o2yTLZHo-qg52tA2VTBMjqJVK_fd7E7ACXL9F18OY/edit#gid=1368925983

...

## How to test the changes?



1. Launch the explorer
2. Verify the different tasks in the spreadsheet

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
